### PR TITLE
Fixes Sending Faxes Using Toner

### PR DESCRIPTION
**Pull request by Twinmold93**
As the code currently stands, whenever you send a fax to Central Command
or the Syndicate, it makes a non-existant copy of the fax using toner
from your own machine. This fix removes unnecessary code and no longer
requires toner to send a fax.

Edit: I changed the sanity check from a positive empty check to a negative cannot do check, so its one line less.

Fixes: #11172

:cl: Twinmold
Fix: No longer costs toner to send a fax to Central Command
/:cl:

### CODE REVIEW

File: code/modules/paperwork/faxmachine.dm [hunk 1]
```diff
@@ -283,34 +283,25 @@ var/global/list/fax_blacklist = list()
...
 
-	var/obj/item/rcvdcopy
-	if(istype(copyitem, /obj/item/paper))
-		rcvdcopy = copy(copyitem)
-	else if(istype(copyitem, /obj/item/photo))
-		rcvdcopy = photocopy(copyitem)
-	else if(istype(copyitem, /obj/item/paper_bundle))
-		rcvdcopy = bundlecopy(copyitem)
-	else
+	if((!istype(copyitem, /obj/item/paper))&&(!istype(copyitem, /obj/item/photo))&&(!istype(copyitem, /obj/item/paper_bundle)))
```
Comment (Fox-McCloud) [inline code/modules/paperwork/faxmachine.dm]: Good heavens almighty, man. Styling.

Comment (Twinmold93) [reply code/modules/paperwork/faxmachine.dm]: I'm not sure what you mean by styling. Just the mere fact it's all compressed or...?

Comment (KasparoVy) [reply code/modules/paperwork/faxmachine.dm]: Oh, nah I think it's that there ain't no spaces around `&&` as well as the unnecessary parentheses around the `!istype()` checks.

Could maybe be written as
`if(!(istype(copyitem, /obj/item/paper) || istype(copyitem, /obj/item/paper_bundle) || istype(copyitem, /obj/item/photo))`

Comment (Twinmold93) [reply code/modules/paperwork/faxmachine.dm]: Just did a change before you posted that. I could just write it that way.

**Comment by Twinmold93** [comment]
For whatever reason, the first time I did the KasparoVy Change, it didn't actually keep the file selected when I submitted it.... I have no idea why.

**Comment by KasparoVy** [comment]
Git funky sometimes

File: code/modules/paperwork/faxmachine.dm (hunk 1 above)
Comment (KasparoVy) [reply code/modules/paperwork/faxmachine.dm]: My bad, I could just about sense you typing away and tried to post it asap :rofl:

**Pull request merged**